utils/cmdline.go

```diff
@@ -70,3 +73,16 @@ func PromptChoice(question string, defaultValue string, allowEmpty, nonInteracti
 		return response, nil
 	}
 }
+
+func MaybeCurlAndExit(req client.AstarteRequest, client *client.Client) {
+	if viper.GetBool("appengine-to-curl") || viper.GetBool("housekeeping-to-curl") || viper.GetBool("pairing-to-curl") || viper.GetBool("realmmanagement-to-curl") {
+		fmt.Println(req.ToCurl(client))
+		os.Exit(0)
+	}
+}
```

Comment:
Instead of duplicating the implementation, why not
```suggestion
func MaybeCurlAndExit(req client.AstarteRequest, client *client.Client) {
	MaybeCurl(req, client)
	os.Exit(0)
}
```

Reply:
That's no good because `MaybeCurlAndExit` would exit even if all `*-to-curl` are false. Using something like
```go
func ShouldCurl() bool {
  return viper.GetBool("appengine-to-curl") || viper.GetBool("housekeeping-to-curl") || viper.GetBool("pairing-to-curl") || viper.GetBool("realmmanagement-to-curl")
}
```
instead of that condition might be a better solution, though.